### PR TITLE
update the version after release

### PR DESCRIPTION
**Pull request by akerekes**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
   <groupId>com.google.cloud.tools</groupId>
   <artifactId>appengine-maven-plugin</artifactId>
-  <version>1.2.2-SNAPSHOT</version>
+  <version>1.3.0-SNAPSHOT</version>
```
Comment (meltsufin) [inline pom.xml]: not 1.2.3?

Comment (akerekes) [inline pom.xml]: we've release 1.3.0-rc1

Comment (loosebazooka) [inline pom.xml]: There's some new features and stuff.

**Comment by loosebazooka** [comment]
@akerekes Update the change log too in this PR if you can.

File: USER_GUIDE.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+You can now deploy the cron/doc/etc. configuration files separately using the new goals:
+
+* `deployCron`
+* `deployDispatch`
+* `deployDos`
+* `deployIndex`
+* `deployQueue`
+
+## Source directory
+_For GAE Flexible projects_ The deployment source directory can be overridden by setting the `appEngineDirectory` parameter in the deploy configuration.
```
Comment (loosebazooka) [inline USER_GUIDE.md]: Does _For_ need to be italicizied?

Comment (akerekes) [inline USER_GUIDE.md]: I want to somehow emphasize, that it's only for flex projects. Maybe a different formatting/wording would be better? I meant it as a separate label, not part of the sentence starting with "The deployment..."

Comment (loosebazooka) [inline USER_GUIDE.md]: Oh right, never mind that makes sense. I was thinking "For _GAE Flexible projects_", but I think what you have makes more sense.

**Pull request merged**